.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: cpp
+compiler:
+    - gcc
+env:
+    matrix:
+        - FIREFOXVERSION="36.0.1"
+        - FIREFOXVERSION="31.5.0esr"
+notifications:
+    email: false
+before_install:
+    - export DISPLAY=:99.0
+    - sh -e /etc/init.d/xvfb start
+    - wget ftp://ftp.mozilla.org/pub/firefox/releases/${FIREFOXVERSION}/linux-x86_64/en-US/firefox-${FIREFOXVERSION}.tar.bz2
```

Comment:
Probably better to use HTTP, to at least give Travis the chance to do intermediate caching. No idea if they do.